### DISCUSSION TITLE
Presets missing audio?

### DISCUSSION THREAD

**Discussion by bxktty** [publicsamples/Oi-Grandad#10] [Q&A]
Hello!! I saw your plugin in [this](https://www.youtube.com/watch?v=PEB5pkHjVeo) venus theory video and got immediately to installing it. I am working on a mac I cannot seem to get the vst3 to work in spite of it being in the right location, and completely allowed. Additionally on the AU install the presets do not come linked with any audio and i cannot seem to find the corresponding audio in the Application Support. Thank you so much

[boxkitty](https://twitter.com/boxkitty1)

**Comment by OneShortMoment** [comment]
You probably missed the additional "Presets & Samples" download (like myself and some other users posting in the comments here...)
Have a look here, where I described how I got it to work (on Windows, but I assume it should be the same on a Mac if you exchange your user folder name for what's appropriate there): 
https://github.com/publicsamples/Oi-Grandad/discussions/13#discussioncomment-4210249
